Lib/fontTools/cffLib/specializer.py

```diff
@@ -822,26 +822,67 @@ def specializeProgram(program, getNumRegions=None, **kwargs):
         default=None,
         help="Number of variable-font regions for blend opertaions.",
     )
+    parser.add_argument(
+        "--font",
+        metavar="FONTFILE",
+        default=None,
+        help="CFF2 font to specialize.",
+    )
+    parser.add_argument(
+        "-o",
+        "--output-file",
+        type=str,
+        help="Output font file name.",
+    )
 
     options = parser.parse_args(sys.argv[1:])
 
-    getNumRegions = (
-        None
-        if options.num_regions is None
-        else lambda vsIndex: int(options.num_regions[0 if vsIndex is None else vsIndex])
-    )
-
-    program = stringToProgram(options.program)
-    print("Program:")
-    print(programToString(program))
-    commands = programToCommands(program, getNumRegions)
-    print("Commands:")
-    print(commands)
-    program2 = commandsToProgram(commands)
-    print("Program from commands:")
-    print(programToString(program2))
-    assert program == program2
-    print("Generalized program:")
-    print(programToString(generalizeProgram(program, getNumRegions)))
-    print("Specialized program:")
-    print(programToString(specializeProgram(program, getNumRegions)))
+    if options.program:
```

Comment:
What's this for?

Reply:
That's the existing cmdline interface, I'm retaining it:
```
% ./fonttools cffLib.specializer 0 10 rlineto 20 0 rlineto            
Program:
0 10 rlineto 20 0 rlineto
Commands:
[('rlineto', [0, 10]), ('rlineto', [20, 0])]
Program from commands:
0 10 rlineto 20 0 rlineto
Generalized program:
0 10 rlineto 20 0 rlineto
Specialized program:
10 20 vlineto
```